<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multinuvens</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="processLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineNewProcess">
          <property name="placeholderText">
           <string>Nome do Processo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNewProcess">
          <property name="text">
           <string>Criar Processo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="buttonDeleteProcess">
          <property name="text">
           <string>Excluir Processo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMigrateProcess">
          <property name="text">
           <string>Migrar Processo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listProcess"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="cloudLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineNewCloud">
          <property name="placeholderText">
           <string>Nome da Nuvem</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNewCloud">
          <property name="text">
           <string>Criar Nuvem</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonDeleteCloud">
        <property name="text">
         <string>Excluir Nuvem</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listClouds"/>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="hostLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineNewHost">
          <property name="placeholderText">
           <string>Nome do Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNewHost">
          <property name="text">
           <string>Criar Host</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="buttonDeleteHost">
          <property name="text">
           <string>Excluir Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMigrateHost">
          <property name="text">
           <string>Migrar Host</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listHosts"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="vmLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineNewVM">
          <property name="placeholderText">
           <string>Nome da VM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNewVM">
          <property name="text">
           <string>Criar VM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="buttonDeleteVM">
          <property name="text">
           <string>Excluir VM</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMigrateVM">
          <property name="text">
           <string>Migrar VM</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="listVMs"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
